addons/website/views/snippets/s_process_steps.xml

```diff
@@ -11,53 +11,54 @@
             </defs>
         </svg>
         <div class="container">
+            <h3 class="text-center">Our process in four easy steps</h3>
             <div class="row g-0">
-                <div class="col-lg-3 s_process_step pt24 pb24">
-                    <svg class="s_process_step_connector" viewBox="0 0 100 20" preserveAspectRatio="none">
+                <div class="col-lg-3 s_process_step position-relative pt24 pb24">
+                    <svg class="s_process_step_connector position-absolute z-index-1" viewBox="0 0 100 20" preserveAspectRatio="none">
                         <path d="M 0 10 L 100 10" vector-effect="non-scaling-stroke"/>
                     </svg>
-                    <div class="s_process_step_icon">
-                        <i class="fa fa-shopping-basket fa-2x mx-auto rounded-circle bg-o-color-1"/>
+                    <div class="s_process_step_number d-flex align-items-center justify-content-center mx-auto rounded-circle bg-primary-light">
+                        <h2 class="mb-0 text-primary text-center fw-normal">1</h2>
                     </div>
-                    <div class="s_process_step_content">
-                        <h2>Add to cart</h2>
-                        <p>Let your customers follow <br/>and understand your process.</p>
+                    <div class="s_process_step_content mt-3 px-3 text-center">
+                        <h3>Add to cart</h3>
+                        <p>Let your customers follow and understand your process.</p>
```

Comment:
Can we reduce the length of this demo text? It easily goes in third line while the other are always 2 lines max